### PR TITLE
Updated bulkFormHandler from jQuery to vanilla Javascript

### PR DESCRIPTION
**Pull request by padmasreegade**
updated bulkFormHandler from jquery to vanilla javascript

### CODE REVIEW

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @padmasreegade!

Have you run the test suite? I just tried it, and I'm getting a number of failures that seem likely to be related to your changes:

```
There were 3 errors:

1) VuFindTest\Mink\BulkTest::testBulkActionLimits
Exception: Failed to get text after 6 attempts.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:691
/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:610
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BulkTest.php:127
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BulkTest.php:379

2) VuFindTest\Mink\FavoritesTest::testBulkEmail
Exception: Failed to set value after 6 attempts.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:589
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FavoritesTest.php:583

3) VuFindTest\Mink\FavoritesTest::testEmailPublicList
Exception: Failed to set value after 6 attempts.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:589
/usr/local/vufind/module/VuFind/src/VuFindTest/Feature/UserCreationTrait.php:94
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FavoritesTest.php:683

--

There were 5 failures:

1) VuFindTest\Mink\BulkTest::testBulkEmail
Element not found: .modal-body .alert-danger index 0
Failed asserting that null is of type object.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:431
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BulkTest.php:140
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BulkTest.php:166

2) VuFindTest\Mink\BulkTest::testBulkExport
Element not found: #format index 0
Failed asserting that null is of type object.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:431
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BulkTest.php:302

3) VuFindTest\Mink\FavoritesTest::testBulkExport
Element not found: #format index 0
Failed asserting that null is of type object.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:431
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FavoritesTest.php:615

4) VuFindTest\Mink\FavoritesTest::testBulkDelete
Element not found: .modal-body .btn.btn-primary index 0
Failed asserting that null is of type object.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:431
/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:523
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FavoritesTest.php:873

5) VuFindTest\Mink\ListItemSelectionTest::testPrepareFavoriteList
Element not found: .createAccountLink index 0
Failed asserting that null is of type object.

/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:431
/usr/local/vufind/module/VuFind/src/VuFindTest/Integration/MinkTestCase.php:523
/usr/local/vufind/module/VuFind/tests/integration-tests/src/VuFindTest/Mink/ListItemSelectionTest.php:394

ERRORS!
Tests: 2759, Assertions: 15287, Errors: 3, Failures: 5, Skipped: 18.
```

I did some manual testing in-browser (using Firefox) and it does indeed seem like bulk functionality is now broken. I'm not seeing any errors in my console, but bulk buttons no longer have any effect.

A simple way to see what I'm seeing is to create an account, add an item to your favorites, and then try to do any bulk action in the favorites list. Is any of this working for you?

File: themes/bootstrap3/js/common.js
Comment (demiankatz) [inline themes/bootstrap3/js/common.js]: This change will cause some feature regressions: it breaks VuFind's built-in internationalization functionality and it replaces a message displayed as part of the page's HTML with a generic browser pop-up message, which is a less pleasant user experience. If this change was necessary to get tests to pass, I think we'll need to figure out why so we can restore the functionality that is otherwise lost. @crhallberg and I will both be in the office tomorrow if it would be helpful to take a look at this together.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @padmasreegade -- see below for some questions/comments about the latest changes.

File: themes/bootstrap3/js/common.js
Comment (padmasreegade) [reply themes/bootstrap3/js/common.js]: It worked without changing the previous code. Reverted to lightbox alert.

**Comment by demiankatz** [comment]
Thanks, @padmasreegade. I'm running the full test suite now and will report results when it completes. I've also requested a review from @crhallberg since his Javascript expertise exceeds mine.

**Review by demiankatz — APPROVED** [review]
All tests are passing and the code makes sense to me, but I'll await input from @crhallberg before pushing the merge button, just in case I'm overlooking anything.

**Review by crhallberg — APPROVED** [review]

**Comment by demiankatz** [comment]
Thanks, @crhallberg and @padmasreegade!

**Pull request merged**
